### PR TITLE
Improve "offline"

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

File: scripts/offline [hunk 1]
```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
+#
+# Ping google.com until successful.
+#
 
 function check_online() {
-  if ! ping -q -c 1 google.com &> /dev/null;
-  then
+  while ! ping -4 -c 1 google.com. &> /dev/null; do
```
Comment (szepeviktor) [inline scripts/offline:7]: - make sure we choose IPv4 (could be changed to `-6`)
- use absolute domain name

File: scripts/offline [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
     echo -e "\033[0;31mStill offline..."
-    check_online
-  else
-    echo -e "\033[0;32mSuccesfully connected!"
-    exit 0
-  fi
+    sleep 1
```
Comment (szepeviktor) [inline scripts/offline:9]: Resolves many problems.

For example
- when domain name resolution is not available
- to allow <kbd>Ctrl+C</kbd>

**Comment by bart-jaskulski** [comment]
@szepeviktor I don't know how I always miss those... (If you have any tips on how to always follow github notification, I would be grateful).

Thank you, for the suggested changes. I've increased `sleep` value and added `exit` code, but I like your neat improvement!

Finally, I merge those changes... after a while :sweat_smile:

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**